ui/src/plot.tsx

```diff
@@ -912,8 +958,10 @@ export interface Visualization {
   name?: S
   /** True if the component should be visible. Defaults to True. */
   visible?: B
-  /** The events to capture on this visualization. */
+  /** The events to capture on this visualization. One of 'select_marks'. */
   events?: S[]
+  /** The interactions to be allowed for this plot. One of 'drag-move' | 'scale-zoom' | 'brush'. */
```

Comment:
Better to use `snake_case` (consistent with how events are named, e.g. `select_marks`) so that users don't have to think when to use which.

Reply:
Agree, good catch.